src/components/generic/TimerDisplay.tsx

```diff
@@ -0,0 +1,42 @@
+import { CalculateTime } from '../../utils/helpers';
+import type { timeProps } from '../../utils/helpers';
+
+/* component details 
+    - reusable component that displays the the appropriate time elements for the stopwatch or countdown timers
+    - takes an object entries and together with map creates the elements and renders the appropriate time
+    - CalculateTime is a function that takes the milliseconds and converts
+*/
+
+interface TimerDisplayProps extends timeProps{
+    flag: boolean;
+    time: object;
+}
+
+export const TimerDisplay = (props: TimerDisplayProps ) => {
+    let objTime: number | string | object = 0;
+
+    if (props.flag) {
+        objTime = CalculateTime(props);
```

Comment:
function naming should be camel**C**ase - only functional components should be **P**ascal**C**ase